src/neo/Wallets/Wallet.cs

```diff
@@ -237,6 +238,21 @@ public WalletAccount GetAccount(ECPoint pubkey)
             return GetAccount(Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash());
         }
 
+        /// <summary>
+        /// Gets the default account of the wallet.
+        /// </summary>
+        /// <returns>The default account of the wallet.</returns>
+        public virtual WalletAccount GetDefaultAccount()
+        {
+            WalletAccount first = null;
+            foreach (WalletAccount account in GetAccounts())
+            {
+                if (account.IsDefault) return account;
```

Comment:
Should we check to ensure there is only a single account where `IsDefualt == true`?

Reply:
No. We just need to return the first one.